Enable auto-apply for run trigger runs (#377)
hashicorp/terraform-docs-common: website/docs/cloud-docs/workspaces/settings/run-triggers.mdx

Under the "Source Workspaces" section, select the workspace you would like to connect as your source and click "Add workspace". You now have a run trigger established with your source workspace. Any run from that source workspace which applies successfully will now cause a new run to be queued in your workspace.

> ~> **Important:** Runs initiated in your workspace as the result of a run trigger connection to a source workspace may auto-apply depending on your [auto-apply-run-trigger](/terraform/cloud-docs/workspaces/settings#run-triggers) setting selection.

Reviewer: I found the message itself softened from "will never auto-apply" to "may auto-apply" but the severity went from "note" to "important"-- I think we should explain the interaction here but I also think there's nothing unexpected that needs a callout box.

```suggestion
## Run Triggers Auto-Apply Setting

Runs initiated by run trigger will not auto-apply unless the [auto-apply-run-trigger](/terraform/cloud-docs/workspaces/settings#run-triggers) setting is enabled. This setting operates independently of the workspace [auto-apply](/terraform/cloud-docs/workspaces/settings#auto-apply-and-manual-apply) setting.
```

Under the "Source Workspaces" section, select the workspace you would like to connect as your source and click "Add workspace". You now have a run trigger established with your source workspace. Any run from that source workspace which applies successfully will now cause a new run to be queued in your workspace.

<!-- BEGIN: TFC:only name:auto-apply-run-trigger -->

## Run Triggers Auto-Apply Setting

> Runs initiated by run trigger will not auto-apply unless the "Auto-apply run triggers" setting is enabled. This setting operates independently of the primary workspace [auto-apply](/terraform/cloud-docs/workspaces/settings#auto-apply) setting.

Reviewer 2: I'm assuming the UI actually says "Auto-apply run triggers"? If so then we can bold it like so:
```suggestion
Runs initiated by a run trigger do not auto-apply unless you enable the **Auto-apply run triggers** setting. This setting operates independently of the primary workspace [auto-apply](/terraform/cloud-docs/workspaces/settings#auto-apply) setting.
```
